domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaycontextparameter/DateTimeDisplayValidatorImpl.java

```diff
@@ -13,6 +13,9 @@
 @Component
 public class DateTimeDisplayValidatorImpl implements DisplayContextParameterValidator {
 
+    Pattern NOT_ALLOWED_CHARACTERS_PATTERN_DATETIME = Pattern.compile("[VzOXxZ]");
```

Comment:
Please follow naming conventions, these variables are not static, final.... Then no need to define them in CAPITAL letters

Reply:
They can be static final now following the move. Adjusted, so no need to rename.